src/robot/client.ts

```diff
@@ -353,32 +389,7 @@ export class RobotClient implements Robot {
            * connection getting closed, so restarting ice is not a valid way to
            * recover.
            */
-          if (this.peerConn?.iceConnectionState === 'closed') {
-            if (this.webrtcOptions?.noReconnect) {
-              return;
-            }
-
-            let retries = 0;
-            // eslint-disable-next-line no-console
-            console.debug('connection closed, will try to reconnect');
-            void backOff(() =>
-              this.connect().then(
-                () => {
-                  // eslint-disable-next-line no-console
-                  console.debug('reconnected successfully!');
-                  events.emit('reconnected', {});
-                },
-                (error) => {
-                  // eslint-disable-next-line no-console
-                  console.debug(
-                    `failed to reconnect - retries count: ${retries}`
-                  );
-                  retries += 1;
-                  throw error;
-                }
-              )
-            );
-          }
+          this.isConnected() ? events.emit('reconnected', {}) : events.emit('disconnected', {});
```

Comment:
Lint error caused by this line:
`392:11  error  Expected an assignment or function call and instead saw an expression`